data/README.md

```diff
@@ -0,0 +1,30 @@
+# Data Guide
+
+For a full explanation of the different data sources please see the [project wiki page](https://github.com/DynamicGenetics/Covid-Communities-Map/wiki).  
+
+## Folder Structure
```

Comment:
I am not sure whether we would really need this parceling of the data folders, although I reckon this would make things easier to find. 

So I am leaving here as a general comment for later thoughts to discuss :)

Reply:
Abosolutely! The intention was for it to be very clear what each file was, and what stage of the pipeline it is from, though this could be done in lots of ways and I'm very happy to make changes on feedback :)

Reply:
makes total sense ;)